stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -10,40 +10,86 @@ This implementation supports introducing all WebSub components: subscribers, pub
 is updated by the topic's publisher.
  
  
-#### Basic Flow with WebSub
+### Basic Flow with WebSub
 1. Subscriber discovers the topics it needs to subscribe to in order to receive updates/content and discovers the hub(s) where it can subscribe.
 
 2. Subscriber sends a subscription request to a hub, specifying the topic it needs to receive notifications for along 
```

Comment:
```suggestion
2. The subscriber sends a subscription request to a hub specifying the topic of which it needs to receive notifications  along 
```

Reply:
IMO "topic of which" does not read right. You subscribe to receive notifications for a particular topic. Thus "the topic it needs to receive notifications for". WDYT?